test/fixtures/errors.html

```diff
@@ -24,7 +24,7 @@
 
 <amp-youtube
     id="yt0"
-    data-expectederror="Exactly one of data-videoid or data-live-channelid should be"
+    data-expectederror="Exactly one of data-videoid, data-live-channelid or data-channelid should be present for <amp-youtube>"
```

Comment:
@banaag would this need a update to the validator?

https://github.com/ampproject/amphtml/blob/e742aa10f22d5366ae37bf7f17a73c7332ecf179/extensions/amp-youtube/validator-amp-youtube.protoascii